team/quiet.rst

```diff
@@ -0,0 +1,40 @@
+
+############
+Quiet Friday
+############
+
+Starting on 2020-09-25 DMLT will commence **Quiet Friday**. 
+
+Guidelines
+==========
+The driver for the quiet day is to allow staff a block of time without interruptions or the need to be online and receptive. 
+By reducing traffic we will reduce FOMO on discussion topics. 
+Hence there should be: 
+
+- No general meetings should be planned. There may of course be exceptions, for example if your focused work includes a co-work with others who agree to meet this is ok. Such agreements should be between peers and not use any hierarchy. You may elect to attend meetings/workshops outside DM - or occasional we may be required to attend such meetings.
```

Comment:
We prefer one line per sentence since that makes commenting easier and github happier.

occasional -> occasionally